src/vscode-bicep/package-lock.json

```diff
@@ -7179,12 +7180,63 @@
         "node": ">=4"
       }
     },
+    "node_modules/json-merger": {
+      "version": "1.1.9",
+      "resolved": "https://registry.npmjs.org/json-merger/-/json-merger-1.1.9.tgz",
+      "integrity": "sha512-kbt/SUtX3sMm11PjBG4CjgjIEaYKqWP7xYk8lwIlXUs1EXCZaIHKv9uqSxAs6EPdisQhsa2k64C3I/RVn+562w==",
+      "dev": true,
+      "dependencies": {
+        "commander": "^9.1.0",
+        "fs-extra": "^10.0.1",
+        "js-yaml": "^4.1.0",
+        "json-ptr": "^3.1.0",
+        "jsonpath": "^1.1.1",
+        "lodash.range": "^3.2.0",
+        "vm2": "^3.9.9"
```

Comment:
Not related to your change, but :thinking: this is a way bigger dependency closure than I would have expected for a `deepmerge` function.

Reply:
Yeah especially vm2!